resources/testdata/cf-sample-app-nodejs.yaml

```diff
@@ -0,0 +1,9 @@
+# From https://github.com/cloudfoundry-samples/cf-sample-app-nodejs/blob/master/manifest.yml
+---
+applications:
+- name: cf-nodejs
+  memory: 512M
+  instances: 1
+  random-route: true
```

Comment:
`random-route` is v2, no?

Reply:
Seems like it's in V3 too:
https://docs.cloudfoundry.org/devguide/deploy-apps/manifest-attributes.html#random-route

Same for `no-route`. The only one that's not there is `default-route` and I had doubts whether to capture it or not, since by default the CF controller will create a route with the app name, unless any of the other route fields are defined. I leave it up to the user to interpret this behavior rather than us.